src/openforms/formio/api/validators.py

```diff
@@ -76,6 +77,14 @@ def __call__(self, value: UploadedFile) -> None:
                 _("The provided file is not a valid file type.")
             )
 
+        if len(file_name_parts) == 1:
+            raise serializers.ValidationError(
+                _(
+                    "Could not determine the file type. Please make sure the file name "
+                    "has an extension."
+                )
+            )
```

Comment:
```suggestion
        if not ext:
            raise serializers.ValidationError(
                _(
                    "Could not determine the file type. Please make sure the file name "
                    "has an extension."
                )
            )
```
this `file_name_parts` check is really a convoluted way to say "I couldn't determine an extension".